### PR TITLE
FIX: Removed dependency of Devtron micro-services on dockerhub

### LINKED ISSUE #498: Remove/Minimize dependency of Devtron micro-services from dockerhub
# Summary

Dockerhub pull rate limit sometimes interferes in the installation/recovery of Devtron micro-services if the pull rate limit is hit.  

# Motivation

Dockerhub pull rate limits are very low and often gets hit if the Devtron cluster is running on spot nodes and dramatically increases the recovery time.

# Proposal

Replace all the dockerhub dependent images with their corresponding official images in quay or push the images into devtron quay and use them in the manifests

### PR DESCRIPTION
**Pull request by ansh-mittal01**
Fixes #498 
Replaced all the dockerhub dependent images with their corresponding official images in quay or pushed the images into Devtron quay and used them in the manifests as suggested.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested by installing Devtron using Helm3

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented on my code, particularly in hard-to-understand areas
* [x] I have tested it for all user roles

### CODE REVIEW

File: manifests/yamls/argocd.yaml [hunk 1]
```diff
@@ -2437,7 +2437,7 @@ spec:
       - command:
         - /shared/argocd-util
         - rundex
-        image: quay.io/dexidp/dex:v2.25.0
+        image: "quay.io/dexidp/dex:v2.25.0"
         imagePullPolicy: Always
```
Comment (prakarsh-dt) [inline manifests/yamls/argocd.yaml:2441]: Let's also make `imagePullPolicy: IfNotPresent` wherever it is set `Always` that will un-necessary download of the image.

Comment (ansh-mittal01) [reply manifests/yamls/argocd.yaml]: Okay

File: manifests/yamls/grafana.yaml [hunk 2]
```diff
@@ -520,7 +520,7 @@ spec:
             - name: sc-dashboard-volume
               mountPath: "/tmp/dashboards"
         - name: grafana
-          image: "grafana/grafana:7.3.1"
```
Comment (prakarsh-dt) [inline manifests/yamls/grafana.yaml:523]: Please use grafana standard image not bitnami packaged grafana

**Review by pawan-59 — COMMENTED** [review]
@prakarsh-dt please check

File: manifests/yamls/workflow.yaml [hunk 3]
```diff
@@ -253,6 +253,6 @@ spec:
         - argoproj/argoexec:v2.3.0
```
Comment (prakarsh-dt) [inline manifests/yamls/workflow.yaml:253]: Replace argoproj/argoexec:v2.3.0 image too.

**Review by prakarsh-dt — APPROVED** [review]

**Review by pghildiyal — APPROVED** [review]

**Pull request merged**